src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
Most coding standards are just equally easy to follow. (Oh, hey, nice, a tab lover; we seem to be rare in this PHP ecosystem… :-D)

But well, it's all subjective… the only thing I definitely find weird in the inconsistent bracing style … if it were all new-line braces or none … fine… but this is too weird for me to follow when writing code (yea, I know, autoformat, but having to apply it each time is annoying … and it conflicts with the OCD every sane programmer has).